src/main/java/model/input/file/DataFile.java

```diff
@@ -144,7 +150,35 @@ public static Class getColumnType(String type) throws ClassNotFoundException {
 				throw new ClassNotFoundException();
 		}
 	}
-	
+
+	/**
+	 * Returns a string describing the type of the class.
+	 * @param type The class
+	 * @return The type of class as a string
+	 * @throws ClassNotFoundException
+	 */
+	public static String getStringColumnType(Class type) throws ClassNotFoundException {
+		if (type == IntValue.class) {
+			return "int";
+		}
+		if (type == FloatValue.class) {
```

Comment:
Is het niet beter om in datavalue een abstract getTypeString() te doen? Dan is deze hele switchcase niet nodig